lib/src/native_updater.dart

```diff
@@ -19,41 +25,70 @@ class NativeUpdater {
   /// Checking New Version
   static displayUpdateAlert({
     @required BuildContext context,
-    @required bool requiresUpdate,
+    @required bool forceUpdate,
     String appStoreUrl,
+    String titlePrefix,
+    String description,
+    String updateButtonLabel,
+    String closeButtonLabel,
+    String ignoreButtonLabel,
   }) async {
     /// Get Current installed version of app
     final PackageInfo info = await PackageInfo.fromPlatform();
 
-    if (requiresUpdate) {
-      _singleton._appName = info.appName;
-      _singleton._packageName = info.packageName;
-      _singleton._appStoreUrl = appStoreUrl;
-      _singleton._showUpdateAlertDialog(context);
-    }
+    _singleton._forceUpdate = forceUpdate;
+    _singleton._appName = info.appName;
+    _singleton._packageName = info.packageName;
+    _singleton._appStoreUrl = appStoreUrl;
+    _singleton._titlePrefix = titlePrefix;
+    _singleton._description = description;
+    _singleton._updateButtonLabel = updateButtonLabel;
+    _singleton._closeButtonLabel = closeButtonLabel;
+    _singleton._ignoreButtonLabel = ignoreButtonLabel;
```

Comment:
@JeffryHermanto, why we should set it to singleton? Because it's filled in the function level instead of the constructor level. We can directly use the params instead. Do you have any concerns about doing this way?

Reply:
To provide a global point of access to the object.

Reply:
I think it would make sense if you use the constructor just like your `UpgradeMaterialAlert` and `UpgradeCupertinoAlert`.

But I'm still don't get it why you need the `global point of access to the object` :pensive:

Reply:
As discussed, I know that you need it on the other private function in this class